Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/base_xeno.yml

```diff
@@ -342,7 +342,7 @@
     durationMultiplier: 0.666
   - type: RMCCanBeFultoned
   - type: CMArmor
-    explosionArmorStep: 10
+    explosionArmor: 10
```

Comment:
This just isn't supposed to be here at all, so it doesn't have a gameplay change in the end